### PR TITLE
json endpoint active

### PR DESCRIPTION
**Pull request by joooooooooe-star**
adds very crude json acceptance for endpoint

### CODE REVIEW

File: backend/main.py [hunk 1]
```diff
@@ -1,11 +1,73 @@
...
+        'dh': int
+        'speed': int
+        'ten': int
+        'pie': int
+    'comp': Array
+    """
+    data = request.get_json()
+
+    if not data:
+        return "abort", abort(400)
```
Comment (joooooooooe-star) [inline backend/main.py:27]: this is probably wrong

File: backend/main.py [hunk 2]
```diff
@@ -1,11 +1,73 @@
...
+
+    # Convert job strings to enum
```
Comment (joooooooooe-star) [inline backend/main.py:29]: i made a useless comment

File: backend/pps/sch.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class SchPps:
```
Comment (ackao) [inline backend/pps/sch.py:1]: accel's thing got merged in, can use that now

File: backend/main.py [hunk 4]
```diff
@@ -1,11 +1,73 @@
...
+    job_string_to_enum = {
```
Comment (ackao) [inline backend/main.py]: pull this out into a separate helper function, we'll want this for other stuff i assume

**Review by ackao — CHANGES_REQUESTED** [review]

File: backend/pps/sch.py (hunk 3 above)
Comment (ackao) [reply backend/pps/sch.py]: oh hey i was able to just merge it in, ez

File: backend/main.py (hunk 4 above)
Comment (ackao) [reply backend/main.py]: god damn it i looked at an outdated version

**Review by ackao — APPROVED** [review]

**Pull request merged**